.github/workflows/scan-python.yaml

```diff
@@ -29,7 +29,7 @@ on:
       uv-export-extra-args:
         required: false
         type: string
-        default: ""
+        default: '["--all-extras"]'
```

Comment:
Given how we're using this, what about just making it be a multi-line string where each line is a set of args?

Reply:
What would be the pros or your suggestion?

Because otherwise I think I prefer exposing a consistent interface to users of the workflow, so to have a single way to provide lists, especially since we are already bending a bit the way to use the "string" type of the workflow and we already have to remember the correct way to provide this list for other inputs.

Reply:
Using multi-line strings like this is already what other plugins like [setup-python](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#specifying-multiple-pythonpypy-versions) do when the user wants to enter multiple versions of something. Using the JSON version was the only way though to let it become an array in the final output though rather than a string.

Also, by using multi-line strings we're backwards compatible.